build-samples/build_samples.sh

```diff
@@ -10,9 +10,9 @@ KERNEL="$1"
 TOOLCHAIN="$2"
 export KBUILD_OUTPUT="$3"
 
-LLVM_VER="$(llvm_version $TOOLCHAIN)" && :
-if [ $? -eq 0 ]; then
-	export LLVM="-$LLVM_VER"
+if [ $TOOLCHAIN = "llvm" ]; then
```

Comment:
same comment as above.
I believe
```
export LLVM="-${LLVM_VERSION}"
```

is enough.

We should probably add a comment explaining what LLVM is used for (see comment above) as it is non-trivial otherwise.